client/src/App.jsx

```diff
@@ -35,25 +51,35 @@ class App extends React.Component {
     cb(itemId);
   }
 
+
+
   render () {
+    window.addEventListener('click', function (e) {
+      let time = new Date().toLocaleString();
+      let body = {
+        time: time,
+        module: findId(e.target.parentNode),
+        element: e.target.outerHTML
+      };
+      console.log('this is your click event', body);
```

Comment:
leaving this for testing?

Reply:
yes, until we have the axios post that goes right after this and i wanted yall to see what the click events actually register so we can use that ifo in the proper params for post to api.